atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -131,7 +131,9 @@ public TokenRangeWritesLogger getTokenRangeWritesLogger() {
             tokenRangeWritesLogger.updateTokenRanges(tokenMap.asMapOfRanges().keySet());
             return servers;
         } catch (Exception e) {
-            log.error("Couldn't grab new token ranges for token aware cassandra mapping!", e);
+            log.info("Couldn't grab new token ranges for token aware cassandra mapping. We will retry in {} seconds.",
```

Comment:
This one's unclear to me - whether it should be warn or info

Reply:
The exception is thrown when:
* The C* call to get the token ring fails
* Something is borked and we invent a host that is not in the config (shouldn't happen)

Since we retry every 120 seconds by default, it seems very unlikely to me that any engineer will need to take action as a result of the log message, and therefore `INFO` seems appropriate to me.